Header header

time stamp

bool     motion_detected
box      roi

multiBox     rect_facesDetected
multiBoxWTag facesRecog              # recog one face is in service

multiBox     peopleDetected
multiBoxWTag peopleRecog            # people tagged with position updated 